Formula/mplayer.rb

```diff
@@ -31,6 +32,7 @@ class Mplayer < Formula
   depends_on "yasm" => :build
   depends_on "fontconfig"
   depends_on "freetype"
+  depends_on "jpeg-turbo"
```

Comment:
I usually like dependencies to be explicit, but in this case it might be good to omit it to make sure it always uses the same `libjpeg` implementation as `imlib2`.

Maybe it's the `imlib2` dependency that should be made explicit instead.

Reply:
`imlib2` was actually indirect via `libcaca` and configure only has `--enable-caca`.

`jpeg-turbo` is a direct feature with configure option `--enable-jpeg`.